<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nuevo</class>
 <widget class="QDialog" name="nuevo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre del archivo:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnombrearchivo_nuevo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>281</width>
     <height>27</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="maxLength">
    <number>26</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Agregar Campos&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre de Campo:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnombrecampo_nuevo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>281</width>
     <height>27</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>30</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combotipo_nuevo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>281</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Char</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Entero</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Tamaño:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spintamano_nuevo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="bagregar_nuevo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>290</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tablacampos_nuevo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>401</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Nombre</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tipo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tamaño</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>EsLlave</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="bcrear_nuevo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>530</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Crear</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Es llave:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combollave_nuevo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>No</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sí</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
